server/playerServer/database.sql

```diff
@@ -58,6 +66,12 @@ CREATE TABLE IF NOT EXISTS regions (
     INDEX (owner, maxX, minX, maxY, minY, room)
 );
 
+CREATE TABLE IF NOT EXISTS regions_permissions (
+    regionId INT UNSIGNED,
+    allowedUser INT UNSIGNED,
```

Comment:
I'd rather have this be called userId since in the future we want to add multiple permissiontypes.